libs/spark/CHANGELOG.md

```diff
@@ -2,6 +2,40 @@
 
 ## [vNext](https://github.com/prenda-school/prenda-spark/compare/v1.0.0-alpha.3...vNext) (yyyy-mm-dd)
 
+### Unstable Preview
+
+_This section details previews of breaking changes or experimental features that are subject to breaking changes at any time._
+
+- **Unstable_SvgIcon**
+  - Initial implementation of `SvgIcon` replacement according to PDS v2.
+  - Supports rendering _without_ `theme` being in an ancestor `ThemeProvider`.
+  - CSS API Changes:
+    - Removed all class keys _except `"root"`_.
+  - Props API Changes:
+    - `classes`
+      - Removed all class keys _except `"root"`_.
+    - `color`
+      - Removed values: `"dark"`, `"light"`, `"white"`, `"disabled"`, `"error"`, `"success"`, `"warning"`, `"info"`.
```

Comment:
With the error/success/warning/info removed, is there a way to use these in notices that have these settings?

Reply:
or do we now have to be explicit?

Reply:
Or from what we did today it will inherit what color it needs to be, right?